x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/compliance_dashboard.tsx

```diff
@@ -308,32 +318,46 @@ export const ComplianceDashboard = () => {
       selectedTab,
       hasFindingsKspm,
       hasFindingsCspm,
+      isKspmInstalled,
+      isCspmInstalled,
     ]
   );
 
   return (
-    <CloudPosturePage query={selectedTab === 'cspm' ? getCspmDashboardData : getKspmDashboardData}>
-      <EuiPageHeader
-        bottomBorder
-        pageTitle={
-          <CloudPosturePageTitle
-            title={i18n.translate('xpack.csp.dashboard.cspPageTemplate.pageTitle', {
-              defaultMessage: 'Cloud Security Posture',
-            })}
-          />
-        }
-        tabs={tabs.map(({ content, ...rest }) => rest)}
-      />
-      <EuiSpacer />
-      <div
-        data-test-subj={DASHBOARD_CONTAINER}
-        css={css`
-          margin-left: auto;
-          margin-right: auto;
-          height: 100%;
-        `}
-      >
-        {tabs.find((t) => t.isSelected)?.content}
+    <CloudPosturePage
+      query={
+        selectedTab === 'cspm'
+          ? getCspmDashboardData.data
+            ? getCspmDashboardData
+            : undefined
+          : getKspmDashboardData.data
+          ? getKspmDashboardData
+          : undefined
```

Comment:
Nested ternaries are generally harder to read and understand. In this case, using if/else structure would improve the legibility or you store the logic in a variable.
``` js
const getDashboardDataByPostureTypeTab = (selectPostureTypeTab) => {
      if (postureTypeTab === 'cspm') {
        return getCspmDashboardData.data ? getCspmDashboardData : undefined;
      } else {
        return getKspmDashboardData.data ? getKspmDashboardData : undefined;
      }
}
```